tests/data.sysl

```diff
@@ -42,3 +44,12 @@ ObjectModel:
   !type Order:
     order_id <: int
     customer <: Customer
+
+PrimitiveAliasModel:
+  !type User:
+        id <: uuid
+        acc <: Account
+  !type Account:
+        foo <: string
+  !alias uuid:
+        int
```

Comment:
Missing \n

Reply:
@anzdaddy , updated, can you review again?